docs/topics/mta-6-installing-web-console-on-openshift.adoc

```diff
@@ -226,23 +242,20 @@ The test results indicate that the minimum amount of memory for running {Product
 ** Installing the cluster monitoring tools requires an additional 5 GiB of memory.
 ====
 
-=== Eviction Threshold
-
-A node running applications requires sufficient reserve memory to operate successfully. Whenever an application uses more than its reserved memory allocation, it can trigger an out-of-memory event.
-
-The `--eviction-hard` flag directs the node to evict applications when available memory drops below an absolute value or percentage value.
-
-Set `--eviction-hard` flag along with memory reserves for the node.
+[id="eviction-threshold_{context}"]
+=== Eviction threshold
 
-The following example illustrates node memory allocation settings:
+Each node has a certain amount of memory allocated to it. Some of that memory is reserved for system services. The rest of the memory is intended for running pods. If the pods use more than their allocated amount of memory, it triggers an out-of-memory event (node terminated with a `OOMKilled` error).
 
```

Comment:
Although it uses passive voice, I'd rewrite the last clause as "an out-of-memory event... is triggered." I think "it triggers" is ambiguous and the agent that triggers the event is not relevant to the user. Maybe "MTA triggers".

You might want to remove the parentheses and replace them with a comma, that is "event, node terminated...". If "node terminated..." is the phrase users will see or the event's name in the code, it should be in backticks.

Reply:
Rephrased